fmf/base.py

```diff
@@ -363,6 +364,11 @@ class describing the environment context. By default, the key
         Optional 'decision_callback' callback would be called for every adjust
         rule inspected, with three arguments: current fmf node, current
         adjust rule, and whether it was applied or not.
+
+        Optional 'additional_rules' parameter can be used to specify rules
+        which should be applied in addition to those defined in the data
+        themselves. These additional rules are processed even when the applied
```

Comment:
Not sue I have the context, but `in the data themselves` sounds odd to me
```suggestion
        which should be applied in addition to those defined in the data
        itself. These additional rules are processed even when the applied
```

Reply:
google search about data is/are was fun. But en_US seems to prefer 'is'.

Reply:
But in the end I reworded it, hopefully it now makes more sense

Reply:
Ack, it's better.